.drone.yml

```diff
@@ -2,17 +2,9 @@ workspace:
   base: /var/www/owncloud
   path: apps/notifications
 
-branches: [master, stable10, release*]
+branches: [master, stable10, release*, release/*]
```

Comment:
Note: here in stable10, this line mentions ``stable10`` - that made a conflict to resolve when I was backporting.
In ``core`` we mention all the branches in the ``.drone.yml`` of all the branches, I guess just so that they can all look the same.
So we could forward port ``stable10`` into this line in ``master``

Reply:
seems reasonable and fine with me - looking though forward of having only one branch to maintain